apps/timetracker/providers/gsheet.go

```diff
@@ -88,6 +89,12 @@ func getConfigObject(config interface{}) (GsheetConfig, error) {
 	return c, nil
 }
 
+// CleanTimeProviderConfig ...
+func (m *GsheetConnection) CleanTimeProviderConfig(data interface{}) interface{} {
+	data.(map[string]interface{})["email"] = strings.TrimSpace(data.(map[string]interface{})["email"].(string))
```

Comment:
We should use the connection's config object since it is being parsed when the connection object is created.

Reply:
done